src/edu/csus/ecs/pc2/clics/API202306/CLICSRun.java

```diff
@@ -0,0 +1,99 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.JudgementRecord;
+import edu.csus.ecs.pc2.core.model.Run;
+import edu.csus.ecs.pc2.core.util.JSONTool;
+
+/**
+ * CLICS Judgment Run
+ * Contains information about a Problem
+ * 
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("rtFilter")
```

Comment:
For judgments, it prevents the inclusion of max_run_time in the generated response.  max_run_time is a scalar and would always appear, even if 0.  That would not be spec compliant.  (BTW, I have made some changes since the last push to the PR, I think there was an omission in JudgementService where I forgot to add the exceptProps to the filter for one of the cases.  Next push should add that.  But.. that is what the rtFilter does (rt == run_time, FYI).